vector/src/androidTest/java/im/vector/app/ui/robot/settings/SettingsPreferencesRobot.kt

```diff
@@ -34,8 +33,7 @@ class SettingsPreferencesRobot {
         clickOn(R.string.settings_theme)
         clickDialogNegativeButton()
         clickOn(R.string.font_size)
-        waitUntilActivityVisible<FontScaleSettingActivity> {
```

Comment:
`FontScaleSettingActivity` is never idling according to the espresso logs, for now we can check the visibility of the page contents view instead